flow/project.py

```diff
@@ -711,10 +715,15 @@ class FlowGroupEntry(object):
         commands to execute.
     :type options:
         str
+    :param aggregator:
+        aggregator object associated with the :py:class:`FlowGroup`
+    :type aggregator:
+        :py:class:`aggregator`
     """
-    def __init__(self, name, options=""):
+    def __init__(self, name, options="", aggregator=aggregator.groupsof(1)):
```

Comment:
I didn't think of this in the way you're referring to. But I believe since the DefaultAggregatesStore class always iterate over a project, I think this is the required behaviour.

I'm not sure whether it'd be a problem if we use inherited projects. I'll have to test this once.